tracer/src/Datadog.Trace/Activity/ActivityListener.cs

```diff
@@ -159,7 +159,7 @@ static void CreateCurrentActivityDelegates(Type activityType)
             static void ChangeActivityDefaultFormat(Type activityType)
             {
                 // We change the default ID format to W3C (so traceid and spanid gets populated)
-                if (Activator.CreateInstance(activityType, string.Empty).TryDuckCast<IActivityFormat>(out var activityFormat))
+                if (Activator.CreateInstance(activityType, "operationName").TryDuckCast<IActivityFormat>(out var activityFormat))
```

Comment:
What is `"operationName"` used for? Do we change it later?

Reply:
We don't use it for anything, but I'm trying to avoid an exception that could be thrown when the string is empty. I added comments to clear this up in d6273d9224df28263f2f2ef3a19c4fb0f4cb61f5

Reply:
thanks!